src/devices/video/mc6845.cpp

```diff
@@ -1214,6 +1243,7 @@ void mc6845_device::device_start()
 	save_item(NAME(m_line_address));
 	save_item(NAME(m_cursor_x));
 	save_item(NAME(m_has_valid_parameters));
+	save_item(NAME(m_display_disabled_msg_shown));
```

Comment:
This is not a proper state variable but a debugging artifact.

Reply:
Sure, I remove it from save_items, wonder if the logerrors in question should be turned into LOGxxx instead so they are turned off in production?